.gitignore

```diff
@@ -5,3 +5,4 @@ vendor/
 **/Pulumi.*.yaml
 **/yarn-error.log
 yarn.lock
+.idea/
```

Comment:
Can we drop this since it does not seem pertinent to this PR, or is there some particular reason for its inclusion?

Reply:
I added to the PR since I originally commited it on the PR and then had to remove it and then needed to recreate my environment. Happy to remove it if you want but just for reference, we don't have a standardised .gitignore - maybe this is something we should have